src/cmd/ksh93/bltins/print.c

```diff
@@ -375,9 +375,12 @@ int b_print(int argc, char *argv[], Shbltin_t *context) {
         // with syncing history if -s and -f are used together. History is synced
         // later with histflush() function.
         // https://github.com/att/ast/issues/425
-        if (!sflag && sffileno(outfile) != sffileno(sfstderr)) sfsync(outfile);
+        if (!sflag && sffileno(outfile) != sffileno(sfstderr))
+            if (sfsync(outfile) < 0)
```

Comment:
I think you need to run `bin/style all` on your changes.

Reply:
Done.